src/model/scripts/bpe_training/train.py

```diff
@@ -0,0 +1,77 @@
+from datasets import load_dataset
+from tokenizers import ByteLevelBPETokenizer
+from nlpo3 import segment, load_dict
+import argparse
+import os
+
+SPECIAL_TOKENS = [
+    "<s>",
+    "<pad>",
+    "</s>",
+    "<unk>",
+    "<mask>",
+]
+
+DICT_FILE = f"{os.path.dirname(__file__)}/words_th.txt"
+DICT = "dict"
```

Comment:
I think `DICT_NAME` is better than `DICT` and we could use better word than `dict`.

Reply:
done